src/components/contacts/contacts.css

```diff
@@ -1,46 +1,47 @@
 .contact-form {
-  margin: 0 auto;
+    margin-top: 20px
 }
 
 .form-input input[type='text'] {
-  margin-bottom: 30px;
+    margin-bottom: 30px;
```

Comment:
it was 20 and 40 before. Are you sure? Even more - for a such unclear css-name case :)